app/boards/shields/zodiark/zodiark_right.conf

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2021 Spencer Deven
```

Comment:
The current convention is to leave these files out, and we have the issues w/ local builds documented as a fallback.

Reply:
Both files have been removed, Thanks!